definitions/json/CaseEventToFields.json

```diff
@@ -5666,6 +5677,17 @@
     "PageFieldDisplayOrder": 14,
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "nocRequest",
+    "CaseFieldID": "changeOrganisationRequestField",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "COMPLEX",
+    "PageID": "SingleFormPageWithComplex",
```

Comment:
Any reason this is a string and not a number? I understand that ccd will accept this, but its a move away from convention for this project